framework/test/mocha/common/generatePeerHeader.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+// Initialize global variables
+require('../setup');
+
+const randomstring = require('randomstring');
+
+const testConfig = __testContext.config;
+const random = require('./utils/random');
+
+const generatePeerHeader = function(headers = {}) {
```

Comment:
This is low priority but the name `generateNetworkConfig` or `generateP2PConfig` would be closer to new convention but then the file should be renamed too.